lib/omni_auth/strategies/nbp.rb

```diff
@@ -14,8 +14,8 @@ class Nbp < OmniAuth::Strategies::AbstractSaml
         configure idp_metadata
 
         # Our Service Provider has some configuration options
-        option :certificate, File.read(Settings.omniauth.nbp.certificate)
-        option :private_key, File.read(Settings.omniauth.nbp.private_key)
+        # option :certificate, File.read(Settings.omniauth.nbp.certificate)
```

Comment:
🤔

Reply:
This change needs to be reverted before merging, see https://github.com/openHPI/codeharbor/pull/1326#discussion_r1615206485. Since we have an open thread already, I am closing this one (to remove duplicates).